qt/widgets/common/inc/MantidQtWidgets/Common/QtPropertyBrowser/qtpropertymanager.h

```diff
@@ -103,7 +103,7 @@ class EXPORT_OPT_MANTIDQT_COMMON QtGroupPropertyManager
     : public QtAbstractPropertyManager {
   Q_OBJECT
 public:
-  QtGroupPropertyManager(QObject *parent = nullptr);
```

Comment:
Please put the `nullptr` back in this file.

Reply:
Thanks for spotting that. I'll put them back.